tvtk/pyface/ui/qt4/scene.py

```diff
@@ -394,14 +395,25 @@ def save_to_clipboard(self):
         os.unlink(name)
 
     ###########################################################################
+    # 'event' interface.
+    ###########################################################################
+    def _closed_fired(self):
+        del self.picker
```

Comment:
see similar comment in https://github.com/enthought/mayavi/pull/212/files#r34133840